internal/testing/releasehelper.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Helper tool for creating new releases of the Go CDK.
+// Run from the repository root:
+// $ go run internal/testing/releasehelper.go <flags>
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+// TODO(eliben): add bumpversion flag that bumps version by 1, or to a version
+// given on the command line
+var doReplace = flag.Bool("replace", false, "add replace directives to root")
```

Comment:
These file like they should be commands, not flags. In particular, it doesn't make sense to set both flags to `true`.

Reply:
There could be other flags that can be shared with `-replace`, such as bumping dep version (will be needed for future releases). Currently replace wins over remove, and I can just add an error if both are given. Flags are much simpler to add/tweak than getting into commands

Reply:
Added a check